### PR TITLE
added docs for providers

### PR DESCRIPTION
**Pull request by jaceks2106**

### CODE REVIEW

File: docs/providers/Serviceplatformen.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
+Serviceplatformen primarily exhibits personal data. It is the responsibility of the municipality to ensure that this data is processed in accordance with 
+current legislation and thus the municipality's responsibility to ensure that via the Service Platform only access to the necessary personal data is given 
+for a specific IT system.
+
+For CPR purposes Logic connects to 'PersonBaseDataExtended' service [(LINK)](https://www.serviceplatformen.dk/administration/serviceOverview/show?uuid=e6be2436-bf35-4df2-83fe-925142825dc2). 
+Before Logic connection, client must firstly send the request for service agreement at STS Administration portal at specific environment (Test or Production). 
+It must be done by person who has MOCES certificate within KMD. When service agreement is approved, the client must create the configuration at Logic Console.
```
Comment (base2browns) [inline docs/providers/Serviceplatformen.md]: We shouldn't assume it's KMD.

File: docs/providers/Serviceplatformen.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
+Serviceplatformen primarily exhibits personal data. It is the responsibility of the municipality to ensure that this data is processed in accordance with
```
Comment (base2browns) [inline docs/providers/Serviceplatformen.md]: We should probably state that serviceplatform is only for use by Municipalities or systems acting on their behalf.

File: docs/providers/Serviceplatformen.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
+Serviceplatformen primarily exhibits personal data. It is the responsibility of the municipality to ensure that this data is processed in accordance with 
+current legislation and thus the municipality's responsibility to ensure that via the Service Platform only access to the necessary personal data is given 
+for a specific IT system.
+
+For CPR purposes Logic connects to 'PersonBaseDataExtended' service [(LINK)](https://www.serviceplatformen.dk/administration/serviceOverview/show?uuid=e6be2436-bf35-4df2-83fe-925142825dc2). 
+Before Logic connection, client must firstly send the request for service agreement at STS Administration portal at specific environment (Test or Production).
```
Comment (base2browns) [inline docs/providers/Serviceplatformen.md]: Any chance we can provide a link to the STS Admin portal? Can we also talk about how and when to talk to the ServicePlatform help desk?

File: docs/providers/Serviceplatformen.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
```
Comment (base2browns) [inline docs/providers/Serviceplatformen.md]: Suggest we link to https://www.serviceplatformen.dk/

File: docs/providers/Serviceplatformen.md (hunk 2 above)
Comment (jaceks2106) [reply docs/providers/Serviceplatformen.md]: This text was copy-pasted from their page :D

File: docs/providers/Serviceplatformen.md (hunk 4 above)
Comment (jaceks2106) [reply docs/providers/Serviceplatformen.md]: done

File: docs/providers/Serviceplatformen.md (hunk 1 above)
Comment (jaceks2106) [reply docs/providers/Serviceplatformen.md]: Removed info about KMD

File: docs/providers/Serviceplatformen.md (hunk 3 above)
Comment (jaceks2106) [reply docs/providers/Serviceplatformen.md]: Done

**Comment by jaceks2106** [comment]
@base2browns look now

**Pull request closed without merging**